integration/spring-data/base/src/main/java/com/blazebit/persistence/spring/data/base/query/AbstractPartTreeBlazePersistenceQuery.java

```diff
@@ -67,6 +70,12 @@
  *     select item aliases
  *   - For entity attribute sorts we just use the order property as is
  *
+ * Sorters are retrieved from 4 sources. The order these sources are applied to the final query is determined by their
+ * dynamicity, i.e. dynamic sources have precedence over static sources:
+ *   1. {@link EntityViewSettingProcessor} method parameter that adds attribute sorters to the entity view settings
```

Comment:
Pageable/Sort should have the highest precedence. After that, the sorting from settings processor and finally the sorting via repository method name should be applied.